packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -162,6 +163,7 @@ const Dashboard = () => {
         showRedirectButton: true,
     });
     const { mutateAsync: deleteDashboard } = useDeleteMutation();
+    const { mutate: exportDashboard } = useExportDashboard();
```

Comment:
Since this request can take more than a couple of seconds, it would be good to have a loading state.